src/common/Page.ts

```diff
@@ -748,6 +749,10 @@ export class Page extends EventEmitter {
     return this._accessibility;
   }
 
+  get isDragInterceptionEnabled(): boolean {
```

Comment:
It's surprising/unexpected that this is a getter. Other `page.is*` APIs are methods. Patch: https://github.com/puppeteer/puppeteer/pull/7419

Reply:
I looks like this has already been addressed in https://github.com/puppeteer/puppeteer/pull/7419